pkg/cli/admin/release/mirror.go

```diff
@@ -213,6 +370,22 @@ func (o *MirrorOptions) Run() error {
 		return fmt.Errorf("--skip-release-image and --to-release-image may not both be specified")
 	}
 
+	if o.ApplyReleaseImageSignature {
+		if len(o.ReleaseImageSignatureToDir) != 0 {
+			return fmt.Errorf("--release-image-signature-to-dir and --apply-release-image-signature may not both be specified")
+		}
+	} else {
+		if len(o.ReleaseImageSignatureToDir) == 0 {
+			if len(o.ToDir) == 0 {
+				return fmt.Errorf("if --to-dir and --apply-release-image-signature are not specified must use --release-image-signature-to-dir to specify a directory to export signature to")
```

Comment:
change 
`if --to-dir and --apply-release-image-signature are not specified must use --release-image-signature-to-dir to specify a directory to export signature to`  with

 `if --to-dir and --apply-release-image-signature are not specified, --release-image-signature-to-dir must be use to specify a directory to export the signature`

Reply:
I 100% agree but followed the established pattern in the code to be consistent. Not only in mirror but throughout oc it uses "may not" in it's error output.

Reply:
Changed in commit e3f50c1